content/events/2024-10-24-egu2025/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: European Geosciences Union (EGU) 2025
+date: '2024-10-24'
```

Comment:
I guess the date needs to be the one from the event ...

Reply:
yes you're right !